src/Workspaces/CSharp/Portable/CodeGeneration/CSharpSyntaxGenerator.cs

```diff
@@ -2384,7 +2386,6 @@ internal static BaseParameterListSyntax GetParameterList(SyntaxNode declaration)
                 SyntaxKind.DestructorDeclaration => ((DestructorDeclarationSyntax)declaration).ParameterList,
                 SyntaxKind.IndexerDeclaration => ((IndexerDeclarationSyntax)declaration).ParameterList,
                 SyntaxKind.ParenthesizedLambdaExpression => ((ParenthesizedLambdaExpressionSyntax)declaration).ParameterList,
-                SyntaxKind.SimpleLambdaExpression => SyntaxFactory.ParameterList(SyntaxFactory.SingletonSeparatedList(((SimpleLambdaExpressionSyntax)declaration).Parameter)),
                 SyntaxKind.LocalFunctionStatement => ((LocalFunctionStatementSyntax)declaration).ParameterList,
```

Comment:
this was an internal method only used for certain features that legitimately wanted the real parameter list.  I've audited them to ensure they're ok with this change.

Reply:
The features are:
1. assign out params.   I just added this, and it never applies to a single-param lambda since those aren't ever 'out'
2. wrap-parameters.  Never applies to single-param lamdbas as there's only one thing to wrap.
3. GEnerate constructor.  Constructors always have real parameter lists.